client/my-sites/importer/file-importer.jsx

```diff
@@ -93,7 +93,9 @@ class FileImporter extends PureComponent {
 			 *
 			 * This is used for the new Migration logic for the moment.
 			 */
-			cardProps.href = overrideDestination.replace( '%SITE_SLUG%', site.slug );
+			cardProps.href = overrideDestination
+				.replace( '%SITE_SLUG%', site.slug )
```

Comment:
Minor note: should `site.slug` not be URL-encoded?
```suggestion
				.replace( '%SITE_SLUG%', encodeURIComponent( site.slug ) )
```

Reply:
Good question.
Researching I found it is coming from the store > backend.

I didn't find an example of usage that we are encoding it and have concerns about encoding and breaking our API endpoints that are using the slug.

Do you think we should have another task to investigate the siteSlug usage?  I don't have a strong opinion about.

Reply:
Yes, this is not a blocker since, at least, it's not breaking a new thing. We can improve on this later.